### PR TITLE
[Feature](sink) support parallel outfile

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes
support parallel outfile

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by yiguolei — COMMENTED** [review]
Great work

**Comment by BiteTheDDDDt** [comment]
run buildall

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -1478,7 +1478,7 @@ public class Config extends ConfigBase {
      * Whether to allow the outfile function to export the results to the local disk.
      */
     @ConfField
-    public static boolean enable_outfile_to_local = false;
+    public static boolean enable_outfile_to_local = true;
```
Comment (HappenLee) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:1481]: why change the default config？

**Review by HappenLee — DISMISSED** [review]
LGTM

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by Mryange — APPROVED** [review]

**Pull request merged**